### ISSUE TITLE
Unable to set kube-proxy conntrack maxPerCore with extra-config

### ISSUE THREAD

**Issue by dogbertai** [kubernetes/minikube#11103]
I'm trying to set conntrack maxPerCore to 0 in kube-proxy.

## Expected
According to https://kubernetes.io/docs/reference/config-api/kube-proxy-config.v1alpha1/#kubeproxy-config-k8s-io-v1alpha1-KubeProxyConntrackConfiguration

/var/tmp/minikube/kubeadm.yaml should look like
```
apiVersion: kubeproxy.config.k8s.io/v1alpha1
kind: KubeProxyConfiguration
clusterCIDR: "10.244.0.0/16"
metricsBindAddress: 0.0.0.0:10249
conntrack:
  maxPerCore: 0
```

But I cannot get this to happen.

## Actual
minikube start --driver none --extra-config kubeadm.ignore-preflight-errors=SystemVerification --extra-config kube-proxy.conntrack.maxPerCore=0

Results in
cat /var/tmp/minikube/kubeadm.yaml
```
apiVersion: kubeproxy.config.k8s.io/v1alpha1
kind: KubeProxyConfiguration
clusterCIDR: "10.244.0.0/16"
metricsBindAddress: 0.0.0.0:10249
conntrack.maxPerCore: "0"
```
minikube kubectl -- get configmap -n kube-system kube-proxy -o yaml
```
    conntrack:
      maxPerCore: null
      min: null
      tcpCloseWaitTimeout: null
      tcpEstablishedTimeout: null
```

I've tried things like
--extra-config kube-proxy.conntrack-max-per-core=0
--extra-config kube-proxy.conntrack:maxPerCore=0
--extra-config kube-proxy.conntrack={maxPerCore:0}

Using  minikube v1.19.0 on Ubuntu 20.10 (lxc/amd64)

**Comment by spowelljr** [comment]
Hi @dogbertai, I believe this PR fixed your issue (#11419), it will be out in the v1.21.0 release. I'm going to close this issue, if that PR did not fix your issue, feel free to reopen this.

**Comment by xamino** [comment]
Hi, so I'm encountering this issue too which would be fixed with the next release. Since it doesn't seem like the fix will be out soon (milestone says ~40% currently) and being able to set the parameter would be the workaround, any help in getting this to work? I did try it but can't find the config under `/var/tmp/minikube/kubeadm.yaml` to check (I assume it is not on my root system).
